x-pack/plugins/enterprise_search/public/applications/__mocks__/kea.mock.ts

```diff
@@ -110,8 +111,10 @@ export class LogicMounter {
   };
 
   // Automatically reset context & mount the logic file
-  public mount = (values?: object) => {
+  public mount = (values?: object, props?: object) => {
     this.resetContext(values);
+    if (props) this.logicFile.build(props);
```

Comment:
TIL

Reply:
I got that from Kea's docs here for context: https://kea.js.org/docs/guide/advanced#mounting-and-unmounting
```
// build the logic with props (`logic(props)` is short for `logic.build(props)`)
```